### DISCUSSION TITLE
Owners of formula1db.com

### DISCUSSION THREAD

**Discussion by AntonioSchez32** [f1db/f1db#28] [General]
Hi,

I would like to ask about the owners of formula1db.com website. Is there somebody who knows the owner of the website?
@marceloverdijk

Thanks!

**Comment by marceloverdijk** [comment]
Hi @AntonioSchez32 , I have no idea who the owners are.
Note that I'm also working on a website that will be available on www.f1db.com soon ;-)

So just to be clear, formula1db.com is in no way affiliated with F1DB.

**Reply by AntonioSchez32** [reply]
OK, thanks @marceloverdijk . I just asked because in my final degree project I'm using your database in Power BI to do the main part of the model (dimension and fact tables) and visualization, but I want lap by lap position and laptimes that I have already scrapped, but there is no email or form to contact them so they can allow the use of that data.

I have no intention of getting profit, but I want them to give permission to use the data.

**Reply by marceloverdijk** [reply]
Hi @AntonioSchez32 I'm very interested in what you build in your final degree project.
If there is something you can share I would be very interested. This could also be [privately](mailto:marceloverdijk@gmail.com) if you prefer.
